### ISSUE TITLE
keyframes-name-pattern false positive with string interpolation

### ISSUE THREAD

**Issue by jdufresne** [stylelint-scss/stylelint-config-standard-scss#16]
test.scss

```scss
$frames: ("one", "two", "three");

@each $name in $frames {
  @keyframes frame-#{$name} {
    /* CSS ... */
  }
}
```

```
$ npx stylelint test.scss

test.scss
 4:14  ✖  Expected keyframe name to be kebab-case  keyframes-name-pattern
```

The name is kebab case, but due to the string interpolation, the line is flagged as an error.

**Comment by jeddy3** [comment]
@jdufresne Please [open a bug issue in the main stylelint repo](https://github.com/stylelint/stylelint/issues/new?assignees=&labels=&template=REPORT_A_BUG.yml) so that we can fix this upstream in Stylelint itself.

**Comment by jdufresne** [comment]
Thanks. Done at:

https://github.com/stylelint/stylelint/issues/5997
